.github/CONTRIBUTING.md

```diff
@@ -31,42 +42,18 @@ Here are a few things you can do that will increase the likelihood of your pull
 Draft pull requests are also welcome to get feedback early on, or if there is something blocking you.
 
 - Create a branch with a name that identifies the user and nature of the changes (similar to `user/branch-purpose`)
-- Open a pull request
 
 **Commit Structure**
-All the Herbs organization libraries uses seemantic release and commitizen to :
+All the Herbs organization libraries uses [seemantic release](https://www.npmjs.com/package/semantic-release) and commitizen to :
 
 - Generate changelog
 - Control version
 - Publish to npm via github actions
-Please use this format for commit messages (https://semver.org/). 
-
-To help with this task we have implemented the commitizen to all libraries, just follow those steps:
-
-- `npm install -g commitizen`
```

Comment:
Why did you removed this? It's has been moved?

Reply:
anybody need to install the commitizen into machine as global package, just use the project commitizen via script npm run commit

Reply:
anybody need to install the commitizen into machine as global package, just use the project commitizen via script npm run commit